src/app/layout.tsx

```diff
@@ -1,4 +1,5 @@
 import './globals.css';
+import Navbar from '../components/Navbar';
```

Comment:
slightly preferable to use `@/components/Navbar` instead of `../components/Navbar`

Reply:
Fixed in [e15b7b8](https://github.com/hack4impact-utk/uw-211/pull/32/commits/e15b7b8b9386269424bea8f4b95551552db09f4b)